services/sqs/src/main/java/software/amazon/awssdk/services/sqs/internal/batchmanager/RequestBatchManager.java

```diff
@@ -67,6 +67,7 @@ protected RequestBatchManager(RequestBatchConfiguration overrideConfiguration,
     public CompletableFuture<ResponseT> batchRequest(RequestT request) {
         CompletableFuture<ResponseT> response = new CompletableFuture<>();
         pendingResponses.add(response);
```

Comment:
I notice when we're creating this set, we're using:
`Collections.newSetFromMap(new ConcurrentHashMap<>());`

For ConcurrentHashMap, it's more standard to use:
`new ConcurrentHashMap<>().newKeySet()`

Reply:
Done